ci/generate_docs.sc

```diff
@@ -52,7 +58,13 @@ def main(release_commits_override: Map[String, String] = Map.empty) = {
   val docsSourceDir = marathonDir/"docs"
 
   buildDocs(docsBuildDir, docsSourceDir, marathonDir)
-  launchPreview(docsBuildDir)
+  if (preview) {
```

Comment:
Why not `val docsDir = buildDocs(...)`? This way one can chain the build much better.